conf/messages.en

```diff
@@ -393,7 +393,6 @@ cannotSendUnloadingRemarks.linkText = View the status of this notification
 departureTransportMeans.identification.heading=Which identification do you want to use for the new departure means of transport?
 departureTransportMeans.identification.title=Which identification do you want to use for the new departure means of transport?
 departureTransportMeans.identification.error.required=Select which identification you want to use for the new departure means of transport
-departureTransportMeans.identification.paragraph=This is the means of transport used from the UK office of departure to a UK port or airport.
```

Comment:
Were you meant to remove this? Perhaps a merge conflict gone wrong?

Reply:
Looks like `TransportMeansIdentificationViewSpec` is missing a check to see if this paragraph is there. Would you be able to add that in?